package.json

```diff
@@ -44,8 +44,8 @@
     "chromatic": "chromatic -t 9q99in2ygnh"
   },
   "peerDependencies": {
-    "react": ">= 17.0.1",
-    "react-dom": ">= 17.0.1",
+    "react": ">= 16.6.1",
+    "react-dom": ">= 16.6.1",
```

Comment:
I think this will throw warnings for users of React 17.
I personally have a side project that uses React 17 and I'm getting this peer dependency warning from grommet for not using the correct version of React.

Reply:
I updated this to be
```javascript
"react": "^16.6.1 || ^17.0.0",
"react-dom": "^16.6.1 || ^17.0.0",
```